website/docs/advanced/builder.md

```diff
@@ -0,0 +1,211 @@
+---
+id: builder
+title: Configure MEV Builder
+sidebar_label: Configure MEV Builder
+---
+
+import {HeaderBadgesWidget} from '@site/src/components/HeaderBadgesWidget.js';
+
+<HeaderBadgesWidget commaDelimitedContributors="James" lastVerifiedDateString="May 18th, 2023" lastVerifiedVersionString="v4.0.2" />
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import BuilderPng from '@site/static/img/builder.png';
+
+:::caution
+
+This guide is for advanced Prysm Users to configure their client for the purposes of extracting [MEV](https://ethereum.org/en/developers/docs/mev/). 
+There are risks to using a builder and may result in missed rewards, missed proposals, censored transactions, omitted transactions, and even slashing when used incorrectly. The Prysm team does not provide guidance on which builders/relays are recommended but lists are available for you to make judgement on based on your own values. 
+
+:::
+
+The following guide explains the options to configure the Prysm client to use a [custom builder](https://docs.flashbots.net/flashbots-mev-boost/block-builders) via a [relay](https://docs.flashbots.net/flashbots-mev-boost/relay), a middle ware to connect validators to block builders. This configuration is accomplished through flags and settings used on the validator client as well as the beacon node, and will NOT cover how to run your own relay,builder, mev-boost (a middleware aggregator of relays), please refer to their respective guides to update this info. Prysm will use the builder via the relay to produce execution payloads that hold the transactions that bring the most value to the validator instead of the execution client. The execution client will still be needed for Prysm for the purposes of syncing and fallback incase there is an issue with using the builder. The builder will only be used in the event of a validator proposal.
```

Comment:
```suggestion
The following guide explains the options to configure the Prysm client to use a [custom builder](https://docs.flashbots.net/flashbots-mev-boost/block-builders) via a [relay](https://docs.flashbots.net/flashbots-mev-boost/relay), a middleware to connect validators to block builders. This configuration is accomplished through flags and settings used on the validator client as well as the beacon node, and will NOT cover how to run your own relay, builder, mev-boost (a middleware aggregator of relays), please refer to their respective guides to update this info. Prysm will use the builder via the relay to produce execution payloads that hold the transactions that bring the most value to the validator instead of the execution client. The execution client will still be needed for Prysm for the purposes of syncing and fallback in case there is an issue with using the builder. The builder will only be used in the event of a validator proposal.
```